spyder/plugins/variableexplorer/widgets/namespacebrowser.py

```diff
@@ -116,7 +122,10 @@ def setup(self, check_all=None, exclude_private=None,
                         data=None,
                         minmax=minmax,
                         shellwidget=self.shellwidget,
-                        dataframe_format=dataframe_format)
+                        dataframe_format=dataframe_format,
+                        show_callable_attributes=show_callable_attributes,
+                        show_special_attributes=show_special_attributes
+                        )

```

Comment:
This is poorly indented (and I was the one who did it, so sorry for that). It should be

```python
self.editor = RemoteCollectionsEditorTableView(
    self,
    data=None,
    minmax=minmax,
    shellwidget=self.shellwidget,
    dataframe_format=dataframe_format,
    show_callable_attributes=show_callable_attributes,
    show_special_attributes=show_special_attributes
)
```